### PR TITLE
Make EditOrder command give a warning on duplicate order

### LINKED ISSUE #124: Make EditOrder command give a warning on duplicate order
Make `editOrder` command give a warning to the user, telling that there is a duplicate warning if:
* There is an existing order with the same item name and date
* Both the edited order and duplicate order have pending status

### PR DESCRIPTION
**Pull request by awdse22**
Close #124 

Changes made on a branch extending from PR #158

### CODE REVIEW

File: src/test/java/seedu/sellsavvy/logic/commands/ordercommands/EditOrderCommandTest.java [hunk 1]
```diff
@@ -112,7 +112,9 @@ public void execute_duplicateOrderUnfilteredList_warningGiven() {
         EditOrderDescriptor descriptor = new EditOrderDescriptorBuilder(firstOrder).build();
         EditOrderCommand editOrderCommand = new EditOrderCommand(INDEX_THIRD_PERSON, descriptor);
 
-        String expectedMessage = EditOrderCommand.MESSAGE_DUPLICATE_ORDER_WARNING
+        String warningMessage = String.format(EditOrderCommand.MESSAGE_DUPLICATE_ORDER_WARNING,
+                firstOrder.getStatus().toString().toLowerCase());
```
Comment (FooChao) [inline src/test/java/seedu/sellsavvy/logic/commands/ordercommands/EditOrderCommandTest.java:116]: image
kx made this method is previous commit.

File: src/main/java/seedu/sellsavvy/logic/commands/ordercommands/EditOrderCommand.java [hunk 2]
```diff
@@ -84,7 +84,8 @@ public CommandResult execute(Model model) throws CommandException {
         OrderList orderList = model.getSelectedOrderList();
         String feedbackToUser = !orderToEdit.isSameOrder(editedOrder)
                 && orderList.contains(editedOrder)
-                ? MESSAGE_DUPLICATE_ORDER_WARNING
+                ? String.format(MESSAGE_DUPLICATE_ORDER_WARNING,
+                editedOrder.getStatus().toString().toLowerCase())
```
Comment (FooChao) [inline src/main/java/seedu/sellsavvy/logic/commands/ordercommands/EditOrderCommand.java:88]: image
kx made this method is previous commit.

**Review by FooChao — COMMENTED** [review]
just some nitpicks

File: src/test/java/seedu/sellsavvy/logic/commands/ordercommands/EditOrderCommandTest.java (hunk 1 above)
Comment (awdse22) [reply src/test/java/seedu/sellsavvy/logic/commands/ordercommands/EditOrderCommandTest.java:116]: changed

**Review by FooChao — APPROVED** [review]
LGTM!

**Pull request merged**